doc/src/learn/exchange-integration-faq.md

```diff
@@ -0,0 +1,177 @@
+
+### Do rewards appear as inbound / outbound on-chain transaction?
+    
+Yes, rewards are added to the staking pool through a special system transaction at epoch boundaries.
+
+### How long does it take to get the first reward after staking? How frequently are rewards paid out?
+    
+Rewards are compounded every epoch, and paid out when you withdraw your stake. You must stake for the entire duration of an epoch to receive rewards for that epoch.
+
+### Is there a minimum and maximum staking amount (for validation and delegation)?
+    
+There will be a minimum amount required and a maximum amount allowed, as well as limits on stake changes within an epoch. 
+    
+ * Validation: Requires a high minimum amount of SUI.
+ * Delegation: Minimum planned to be 1 SUI (subject to change). 
+    
+Specific amounts will be provided when available.
+
+### How does slashing work, and what are the penalties?
+    
+There will not be slashing for the principal stake allocated. Instead, validators will get penalized by having fewer future rewards when these get paid out. Rewards that have already been accrued are not at risk.
+
+### Does Sui support on-chain governance or voting?
+    
+On-chain governance is not implemented for Sui. This is not planned for in the near future.
+
+## Where can I find block details?
+
+### How can I retrieve the current block height or query a block by height using a Sui endpoint?
+    
+Sui is [DAG](https://cointelegraph.com/explained/what-is-a-directed-acyclic-graph-in-cryptocurrency-how-does-dag-work)-based, so the block-based view of the transaction history is not always the most direct one. To get the latest transaction, use the Transaction Query API:
+ 
+    ```json
+    {
+      "jsonrpc": "2.0",
+      "id": 1,
+      "method": "sui_getTransactions",
+      "params": [
+        "All",
+        <last known transaction digest>,
+        100,
+        "Ascending"
+      ]
+    }
+    ```
+    
+### How do I query for balance changes?
+    
+The following solution is interim:
+Use the `BalanceChangeEvent` using the event query API. `BalanceChangeEvent` was added in October 2022 in this [PR](https://github.com/MystenLabs/sui/pull/5289). 
+
+### How do I track block generation?
+    
+Sui uses checkpoints. This is still under development, but Sui will implement checkpoints. Checkpoints are like blocks that are created periodically (likely every few minutes), but generated asynchronously instead of on the critical path for execution. Each checkpoint contain all of the confirmed transactions since the previous checkpoint.
+    
+A significant amount of the performance benefits of Sui comes from carefully separating the work required to actually finalize a transaction from bookkeeping work like checkpoint construction. We're experimenting with a number of different generation intervals, and this may change over time as we better understand the traffic patterns.
```

Comment:
```suggestion
A significant amount of the performance benefits of Sui comes from carefully separating the work required to actually finalize a transaction from bookkeeping work, like checkpoint construction. We're experimenting with a number of different generation intervals, and this may change over time as we better understand the traffic patterns.
```